app/views/users/backup_code_setup/index.html.erb

```diff
@@ -1,16 +1,71 @@
-<% self.title = t('forms.backup_code.are_you_sure_title') %>
+<% self.title = t('forms.backup_code.title') %>
 
-<%= render PageHeadingComponent.new.with_content(t('forms.backup_code.are_you_sure_title')) %>
+<%= render PageHeadingComponent.new.with_content(t('forms.backup_code.title')) %>
 
-<% t('forms.backup_code.are_you_sure_desc_html').each do |desc_p| %>
-  <p><%= desc_p %></p>
+<p>
+  <%= t('forms.backup_code.subinfo_html', app_name: APP_NAME) %>
+</p>
+
+<div class="margin-y-4">
+  <div class="grid-row">
+    <% [@codes.first(@codes.length / 2), @codes.last(@codes.length / 2)].each do |section| %>
+      <div class="grid-col-12 tablet:grid-col-6">
+        <% section.each do |code| %>
+          <div class="margin-bottom-2">
+            <code class="text-bold h2 bg-base-lightest">
+              <%= RandomPhrase.format(code, separator: '-') %>
+            </code>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+
+  <%= render AlertComponent.new(type: :warning, class: 'margin-bottom-4') do %>
+    <%= t('forms.backup_code.caution_codes') %>
+  <% end %>
+
+  <div class="margin-top-1">
+    <%= render ClickObserverComponent.new(event_name: 'Multi-Factor Authentication: download backup code') do %>
+      <%= render DownloadButtonComponent.new(
+            file_data: @codes.join("\n"),
+            file_name: 'backup_codes.txt',
+            outline: true,
+          ) %>
+    <% end %>
+    <%= render PrintButtonComponent.new(
+          icon: :print,
+          outline: true,
+          type: :button,
+          class: 'margin-top-2 tablet:margin-top-0 tablet:margin-left-2',
+        ) %>
+    <%= render ClipboardButtonComponent.new(
+          clipboard_text: @codes.join(' '),
+          outline: true,
+          class: 'margin-top-2 tablet:margin-top-0 tablet:margin-left-2',
+        ) %>
+  </div>
+</div>
+
+<%= simple_form_for(:backup_code_accepted_form, url: backup_code_continue_path, method: :patch) do |f| %>
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :backup_code_notice_accepted,
+        as: :boolean,
+        label: t('forms.backup_code.saved'),
+        required: true,
+      ) %>
+  <%= f.submit t('forms.buttons.continue'), class: 'margin-top-2' %>
 <% end %>
 
-<%= button_to(
-      backup_code_setup_path,
-      method: :patch,
-      class: 'usa-button usa-button--big usa-button--wide',
-      form_class: 'margin-y-5',
-    ) { t('forms.buttons.continue') } %>
+<% if in_account_creation_flow? %>
+  <%= render PageFooterComponent.new do %>
+    <%= link_to t('two_factor_authentication.backup_codes.add_another_authentication_option'), authentication_methods_setup_path %>
```

Comment:
Let's follow-up on this separately, but I'm not sure I understand the point of an "Add another authentication method" link here, especially since we already have a page that comes right after this in the flow that suggests to the user to add another MFA method. And this link behaves differently than "Add another authentication option"

Reply:
Ok will do. 
I have the wrong translation there and will update it. FWIW it's in the Figma and from what I gathered and how it's implemented is it's to add another method regardless of Backup Codes being the only MFA.